apps/mobile/src/queries/balance/total-balance.query.ts

```diff
@@ -0,0 +1,52 @@
+import { useGetStacksAddresses } from '@/features/balances/stacks/use-get-stacks-addresses';
+import { AccountId } from '@/models/domain.model';
+import {
+  useBitcoinAccountTotalBitcoinBalance,
+  useWalletTotalBitcoinBalance,
+} from '@/queries/balance/bitcoin-balance.query';
+import { useStxBalance } from '@/queries/balance/stacks-balance.query';
+
+import { Money } from '@leather.io/models';
+import { sumMoney } from '@leather.io/utils';
+
+interface TotalBalance {
+  btcBalance: Money;
+  btcBalanceUsd: Money;
+  stxBalance: Money;
+  stxBalanceUsd: Money;
+  totalBalance: Money;
+}
+
+export function useTotalBalance(): TotalBalance {
+  const stacksAddresses = useGetStacksAddresses();
+  const { availableBalance: stxBalance, fiatBalance: stxBalanceUsd } =
+    useStxBalance(stacksAddresses);
+  const { availableBalance: btcBalance, fiatBalance: btcBalanceUsd } =
+    useWalletTotalBitcoinBalance();
+
+  const totalBalance = sumMoney([btcBalanceUsd, stxBalanceUsd]);
+  return {
+    btcBalance,
+    btcBalanceUsd,
+    stxBalance,
+    stxBalanceUsd,
+    totalBalance,
+  };
+}
+
+export function useAccountTotalBalance(accountId: AccountId): TotalBalance {
+  const stacksAddresses = useGetStacksAddresses(accountId);
+  const { availableBalance: stxBalance, fiatBalance: stxBalanceUsd } =
+    useStxBalance(stacksAddresses);
+  const { availableBalance: btcBalance, fiatBalance: btcBalanceUsd } =
+    useBitcoinAccountTotalBitcoinBalance(accountId);
```

Comment:
The `useBitcoinAccountTotalBitcoinBalance`? That's from @kyranjamie in [this PR](https://github.com/leather-io/mono/pull/520)

We have slightly different approaches in that I call it:
- `useStxBalance` and it can return the balance of one or many stacks addresses

Kyran is using:
- `useWalletTotalBitcoinBalance` for multiple accounts
- `useBitcoinAccountTotalBitcoinBalance` for singular accounts 

We can standardise the naming? I think it's named that way for a reason though now. 

As an aside to this, I have [an open Q on that PR](https://github.com/leather-io/mono/pull/520/files#r1822656476) post merge  about naming of `useWalletTotalBitcoinBalance` 

> I forgot to double check that this calculates for multiple wallets. I think it does as keychains contains all accounts (across wallets)
> 
> Trying to think up a better name as `useWalletTotalBitcoinBalance` is singular. Maybe a simple change to useWallets is good enough?